components/navigable-container/stoppers.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress Dependencies
+ */
+import { keycodes } from '@wordpress/utils';
+
+/**
+ * Module Constants
+ */
+const { UP, DOWN, LEFT, RIGHT, TAB } = keycodes;
+
+function stopBubbling( event ) {
+	// Prevents arrow key handlers bound to the document directly interfering
+	event.nativeEvent.stopImmediatePropagation();
+	event.stopPropagation();
+}
+
+function handleHorizontalArrows( event ) {
+	// Do not preventDefault if inside a textarea or input where the arrow keys should move inside it.
+	if ( ! [ 'textarea', 'input' ].indexOf( event.target.nodeName.toLowerCase() ) ) {
```

Comment:
What about `contenteditable`.

Reply:
I probably came to the conclusion that you did below that no-one would want a contenteditable inside an arrowable region. Having said that, I have no idea why I thought textarea and input were more likely :)